Mathlib/Topology/ContinuousMap/Basic.lean

```diff
@@ -199,6 +204,59 @@ def prodSwap : C(α × β, β × α) := .prodMk .snd .fst
 
 end Prod
 
+section Sum
+variable {X Y Z W : Type*}
+  [TopologicalSpace X] [TopologicalSpace Y] [TopologicalSpace Z] [TopologicalSpace W]
+
+/-- `Sum.inl : X → X ⊕ Y` as a bundled continuous map. -/
+def inl : C(X, X ⊕ Y) where
+  toFun := Sum.inl
+  continuous_toFun := by continuity
```

Comment:
You should be able to delete any `continuous_toFun` field which can be proved with `by continuity`, since `by continuity` is exactly the default parameter for this field.